### PR TITLE
Homework_2 (C# basics)

### PR DESCRIPTION
**Pull request by Maryia-Bychkova**

### CODE REVIEW

File: Maryia_Bychkova_Application/Program.cs [hunk 1]
```diff
@@ -100,4 +101,178 @@ public void Task_7()
...
+    public void Task_1()
+    {
+        Console.WriteLine("Enter the number:");
+        var numberToString = Console.ReadLine();
+        var numberToInt = Convert.ToInt32(numberToString);
+        Console.WriteLine(numberToInt);
+
+        Console.WriteLine("Enter the number:");
+        var numberToDouble = Console.ReadLine();
+        var number1ToInt = Convert.ToInt32(numberToDouble);
```
Comment (dnnyhlwdplc) [inline Maryia_Bychkova_Application/Program.cs]: all numbers? all to int32?
please review that you need to use 4 data types :)

File: Maryia_Bychkova_Application/Program.cs [hunk 2]
```diff
@@ -100,4 +101,178 @@ public void Task_7()
...
+    public void Task_2()
+    {
+        Console.WriteLine("Enter the number:"); 
+        var numberToString = Console.ReadLine();
+        var numberToint = int.Parse(numberToString);
+        Console.WriteLine(numberToint);
+
+        Console.WriteLine("Enter the number:");
+        var numberToDouble = Console.ReadLine();
+        var number1Toint = int.Parse(numberToDouble);
```
Comment (dnnyhlwdplc) [inline Maryia_Bychkova_Application/Program.cs]: and here instead  of int.Parse use double.Parse

File: Maryia_Bychkova_Application/Program.cs [hunk 3]
```diff
@@ -100,4 +101,178 @@ public void Task_7()
...
+        Console.WriteLine(d);
+        // 1) --a = 2-1 = 1
+        // 2) b*2 = 3*2 = 6
+        // 2) --a - b/2= 1-6 = 0
+    }
+    public void Task_7()
+    {
+        int a = 5;
+        int b = 5;
+        int c = 5;
```
Comment (dnnyhlwdplc) [inline Maryia_Bychkova_Application/Program.cs]: why do you need c here?

File: Maryia_Bychkova_Application/Program.cs [hunk 4]
```diff
@@ -100,4 +101,178 @@ public void Task_7()
...
+        bool d = a == b;
+        Console.WriteLine(d);
+
+        int a = 6;
+        int b = 5;
+        int c = 5;
```
Comment (dnnyhlwdplc) [inline Maryia_Bychkova_Application/Program.cs]: and here?

**Review by dnnyhlwdplc — CHANGES_REQUESTED** [review]

File: Maryia_Bychkova_Application/Program.cs (hunk 1 above)
Comment (Maryia-Bychkova) [reply Maryia_Bychkova_Application/Program.cs]: Done
+

File: Maryia_Bychkova_Application/Program.cs (hunk 2 above)
Comment (Maryia-Bychkova) [reply Maryia_Bychkova_Application/Program.cs]: Done
+

File: Maryia_Bychkova_Application/Program.cs (hunk 3 above)
Comment (Maryia-Bychkova) [reply Maryia_Bychkova_Application/Program.cs]: You are absolutely sure, there is no reason!
+

File: Maryia_Bychkova_Application/Program.cs (hunk 4 above)
Comment (Maryia-Bychkova) [reply Maryia_Bychkova_Application/Program.cs]: So as well as in previous case 
+

**Review by dnnyhlwdplc — APPROVED** [review]

**Pull request merged**